web/api.go

```diff
@@ -131,7 +132,7 @@ func main() {
 	server.POST("/get_access_token", func(c echo.Context) error {
 
 		req := new(Access)
-		err := c.Bind(req)
+		err := json.NewDecoder(c.Request().Body).Decode(&req)
```

Comment:
So the Bind() call doesn't work? I'd just followed this [guide](https://echo.labstack.com/guide/binding)

Reply:
It doesn't right now in Master. I can continue to debug in order to use Bind. Though there are many tickets on their repo related to it

Reply:
So is echo just a bad lib? None of the docs seem to work....

Do you wanna try to swap echo with gin?

Reply:
Not a bad idea, gin is the framework that Plaid endorsed. I'll look into it more tonight.